core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -85,9 +88,11 @@ export function withEditor() {
                 $reloadEditorContent: computed<ReloadEditorContent>(() => {
                     return {
                         code: store.pageAPIResponse()?.page?.rendered,
-                        isTraditionalPage: store.isTraditionalPage(),
-                        isEditState: store.isEditState(),
-                        isEnterprise: store.isEnterprise()
+                        isClientReady: store.isClientReady(),
+                        isTraditionalPage: untracked(() => store.isTraditionalPage()),
+                        enableInlineEdit: untracked(
+                            () => store.isEditState() && store.isEnterprise()
+                        )
```

Comment:
This should be tracked, `isEditState` changes when we select `device` or `socialMedia`

Reply:
Done!